pkg/local_object_storage/blobstor/delete.go

```diff
@@ -15,18 +13,25 @@ func (b *BlobStor) Delete(prm common.DeletePrm) (common.DeleteRes, error) {
 			res, err := b.storage[i].Storage.Delete(prm)
 			if err == nil || !errors.As(err, new(apistatus.ObjectNotFound)) {
 				if err == nil {
-					storagelog.Write(b.log,
-						storagelog.AddressField(prm.Address),
-						storagelog.OpField("DELETE"),
-						zap.String("type", b.storage[i].Storage.Type()),
-						zap.String("storage ID", string(prm.StorageID)))
+					logOp(b.log, deleteOp, prm.Address, b.storage[i].Storage.Type(), nil)
```

Comment:
It was `prm.StorageID` before.

Reply:
yes, but it is in the `if prm.StorageID == nil` branch. ok, retuned `prm.StorageID`